### ISSUE TITLE
TypeError: Unexpected keyword argument 'parser_args'

### ISSUE THREAD

**Issue by srossross** [Anaconda-Server/anaconda-client#275]
```
mr_chomps:anaconda.org sean$ anaconda upload --package-type file data.csv 
Using anaconda-server api site https://api.anaconda.org
extracting package attributes for upload ...
[ERROR] 
Traceback (most recent call last):
  File "/Users/sean/anaconda/bin/anaconda", line 9, in <module>
    load_entry_point('anaconda-client==1.1.2', 'console_scripts', 'anaconda')()
  File "/Users/sean/anaconda/lib/python2.7/site-packages/binstar_client/scripts/cli.py", line 82, in main
    description=__doc__, version=version)
  File "/Users/sean/anaconda/lib/python2.7/site-packages/binstar_client/scripts/cli.py", line 64, in binstar_main
    return args.main(args)
  File "/Users/sean/anaconda/lib/python2.7/site-packages/binstar_client/commands/upload.py", line 170, in main
    filename, parser_args=args)
  File "/Users/sean/anaconda/lib/python2.7/site-packages/binstar_client/utils/detect.py", line 96, in get_attrs
    return detectors[package_type](filename, fileobj, *args, **kwargs)
TypeError: <lambda>() got an unexpected keyword argument 'parser_args'
mr_chomps:anaconda.org sean$ 
```

cc @malev

**Comment by dsludwig** [comment]
Fixed in #268

**Comment by srossross** [comment]
Thanks, I was using an older version.
